extensions/swagger-ui/deployment/src/main/java/io/quarkus/swaggerui/deployment/SwaggerUiProcessor.java

```diff
@@ -136,6 +136,7 @@ public void getSwaggerUiFinalDestination(
             byte[] indexHtmlContent = generateIndexHtml(openApiPath, swaggerUiPath, swaggerUiConfig, indexRootPathBuildItem);
             webJarBuildProducer.produce(
                     WebJarBuildItem.builder().artifactKey(SWAGGER_UI_WEBJAR_ARTIFACT_KEY) //
+                            .onlyCopyNonArtifactFiles(true)
```

Comment:
Why this change? Shouldn't it be the default if we generalize it?

Reply:
BTW, it's an open question, you're the one with the most knowledge of this stuff so I'm curious to hear your opinion.

Reply:
Setting `onlyCopyNonArtifactFiles` to default as `true`, could break quarkiverse members that have already updated to the new webjar infrastructure with 2.7.0.CR1.

I (personally) don't believe a .Final should intentionally break extensions.
WDYT?

Reply:
Well, if making it the default makes sense, I prefer we do it now rather than after because it will be far more of a pain to change it later.

Now, I don't know if it makes sense to make it the default - I will let you decide what makes more sense on this - but, if it does, let's do it now.

Reply:
Yep, defaulting to true makes sense, and is easy enough to change if one needs to.
Done.